mullvad-api/src/lib.rs

```diff
@@ -96,27 +132,45 @@ impl ApiEndpoint {
 
         let host_var = read_var("MULLVAD_API_HOST");
         let address_var = read_var("MULLVAD_API_ADDR");
+        let disable_tls_var = read_var("MULLVAD_API_DISABLE_TLS");
 
         let mut api = ApiEndpoint {
             host: API_HOST_DEFAULT.to_owned(),
             addr: SocketAddr::new(API_IP_DEFAULT, API_PORT_DEFAULT),
             disable_address_cache: false,
+            disable_tls: false,
+            force_direct_connection: false,
         };
 
-        if cfg!(feature = "api-override") {
-            match (host_var, address_var) {
-                (None, None) => (),
-                (Some(_), None) => panic!("MULLVAD_API_HOST is set, but not MULLVAD_API_ADDR"),
-                (None, Some(_)) => panic!("MULLVAD_API_ADDR is set, but not MULLVAD_API_HOST"),
-                (Some(user_host), Some(user_addr)) => {
-                    api.host = user_host;
-                    api.addr = user_addr
-                        .parse()
-                        .expect("MULLVAD_API_ADDR is not a valid socketaddr");
-                    api.disable_address_cache = true;
-                    log::debug!("Overriding API. Using {} at {}", api.host, api.addr);
-                }
+        if cfg!(feature = "api-override") && (host_var.is_some() || address_var.is_some()) {
+            use std::net::ToSocketAddrs;
+
+            let scheme;
+            if let Some(tls_var) = disable_tls_var {
+                api.disable_tls = tls_var != "0";
```

Comment:
When shortened from `disable_tls_var` to just `tls_var` I think important information got lost. It's critical to keep the logic correct here, but it's not obvious what `tls_var != "0"` should mean really. You can reuse the same name here (`if let Some(disable_tls_var) = disable_tls_var ...`)